test/index.js

```diff
@@ -18,7 +18,7 @@ suite('nv', () => {
     assert.strictEqual(versions[0].versionName, 'v10')
     assert.strictEqual(versions[0].start.toISOString(), '2018-04-24T00:00:00.000Z')
     assert.strictEqual(versions[0].lts.toISOString(), '2018-10-30T00:00:00.000Z')
-    assert.strictEqual(versions[0].maintenance.toISOString(), '2020-04-01T00:00:00.000Z')
+    assert.strictEqual(versions[0].maintenance.toISOString(), '2020-05-19T00:00:00.000Z')
```

Comment:
This change was necessary to make the tests pass. Didn't investigate further since it's unrelated.

Reply:
Interesting, since this test technically tests the upstream json it is probably they changed it.  Now that you have this feature though, we could totally setup a mock mirror and point to it for the tests!

Reply:
Ah, didn't see this PR earlier - this was extended in https://github.com/nodejs/Release/commit/9bc5275b739fa326d76612da7c6d7859b50cb6c9